CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+All contributions are welcome: ideas, patches, documentation, bug reports, complaints, and even something you drew up on a napkin.
+
+Programming is not a required skill. Whatever you've seen about open source and maintainers or community members  saying "send patches or die" - you will not see that here.
```

Comment:
"community members  saying" <-- two spaces before "saying". nittiest nit.

Reply:
Nice catch, looks like it's a bug in their upstream template too if you want the fame and glory of fixing it: https://github.com/elastic/logstash/blob/master/README.md